Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -654,17 +651,20 @@ - (void)loadBannerInternal:(NSString *)placementId
         [self.bannerPlacementIdToAdSize setObject:size
                                            forKey:placementId];
 
+        // calculate VungleAdSize
+        VungleAdSize *adSize = [self getBannerSize:size];
+        
         // create vungle banner ad
-        VungleBanner *vungleBannerAd = [[VungleBanner alloc] initWithPlacementId:placementId
-                                                                            size:[self getBannerSize:size]];
-
-        vungleBannerAd.delegate = bannerAdDelegate;
+        VungleBannerView *vungleBannerViewAd = [[VungleBannerView alloc] initWithPlacementId: placementId vungleAdSize: adSize];
```

Comment:
Extra space  between `initWithPlacementId:` and `placementId`,  `vungleAdSize:` and  `adSize`? Thanks a lot. :)

Reply:
Updated